data/data/alibabacloud/cluster/dns/privatezone.tf

```diff
@@ -52,7 +56,7 @@ resource "alicloud_pvtz_zone_record" "pvtz_record_api" {
 resource "alicloud_pvtz_zone_record" "pvtz_record_masters" {
```

Comment:
I just referred to [Table 5. Required DNS records
Component](https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-dns-user-infra_installing-bare-metal), if this is not needed in the installer, I can remove them.
![image](https://user-images.githubusercontent.com/32291794/142968238-c252b878-ce4a-4bde-97bf-08dcef09f860.png)

Reply:
Those requirements are for on-prem installations. My assumption is that the alibaba cloud already provides DNS resolution from one VM instance to another via the hostname of the VM. I also assume that the hostname of the VMs will not contain the cluster domain but instead contain a domain name specific to the alibaba cloud, in which case the A records created here would not even be relevant. Are either or both of those assumptions invalid?

Reply:
> My assumption is that the alibaba cloud already provides DNS resolution from one VM instance to another via the hostname of the VM. 

@mtulio @kwoodson As you know, there is no open API for the function of Automatically synchronize hostnames. After the Installer is executed, the user needs to manually set it to the open state (I guess). I am not sure if this requires adding the records of `<master><n>.<cluster name>.<base domain>` in terrform, and whether this will affect the installation process of the installer?
![image](https://user-images.githubusercontent.com/32291794/143192997-cae4a6da-4bce-489e-bf27-1c81db57d48d.png)

Reply:
We are able to proceed without this functionality.

Reply:
@kwoodson Okey, so I remove these records about master. @staebler  And the PR #5413 may be meaningless, close it? or update removed code on #5413?

Reply:
If we don't need these A records, let's close #5413 and open a new PR to remove the A records.

Reply:
New PR #5422 : remove the A records